app-modules/resource-hub/src/Filament/Resources/ResourceHubArticles/Pages/ListResourceHubArticles.php

```diff
@@ -124,10 +122,10 @@ public function table(Table $table): Table
                 TernaryFilter::make('public'),
             ], layout: FiltersLayout::BeforeContent)
             ->recordActions([
-                ViewAction::make(),
-                EditAction::make(),
                 ReplicateAction::make()
```

Comment:
Because this is being asked for here, and because we make it "duplicate" in all other places we use `ReplicateAction` as well, let's take care of most of this once and for all.

Instead of changing the label, modal heading, submit action label, etc. here let's remove that in the individual instances and setup a configureUsing override for this in the Filament service provider.